instance.go

```diff
@@ -286,12 +285,32 @@ func (i *Instance) DetermineSecondaryJournalDirectory() (string, error) {
 	return "", fmt.Errorf("unable to determine secondary journal directory")
 }
 
+// DetermineISCDatFileName returns the filename of the InterSystems DAT files used by the instance
+func (i *Instance) DetermineISCDatFileName() string {
+	switch i.Product {
+	default:
```

Comment:
I generally place the default last and most often seem it that way (although, I suppose that could be confirmation bias).